ubuntu-advantage

```diff
@@ -109,6 +111,26 @@ main() {
             service_disable "$service" "$@"
             ;;
 
+        update-fips)
+            local token="$1"
+            local bypass_prompt=0
+            while true ; do
+               case "$2" in
+                  --yes | -y)
```

Comment:
I think we can drop `--yes`, supporting `-y` is enough. No other command has `--option`.

Reply:
+1

Reply:
Fixed